probe/probe.go

```diff
@@ -302,3 +334,362 @@ func probeNode(c *k8s.Client) {
 		fmt.Println("No kubernetes environment found")
 	}
 }
+
+type KubeArmorProbeData struct {
+	OSImage                 string
+	KernelVersion           string
+	KubeletVersion          string
+	ContainerRuntime        string
+	ActiveLSM               string
+	KernelHeaderPresent     bool
+	HostSecurity            bool
+	ContainerSecurity       bool
+	ContainerDefaultPosture tp.DefaultPosture
+	HostDefaultPosture      tp.DefaultPosture
+}
+
+func isKubeArmorRunning(c *k8s.Client) bool {
+	_, err := getKubeArmorDaemonsets(c)
+	return err == nil
+
+}
+
+func getKubeArmorDaemonsets(c *k8s.Client) (bool, error) {
```

Comment:
```suggestion
func getKubeArmorDaemonset(c *k8s.Client) (bool, error) {
```
I believe there would be just one daemonset in a cluster so...